justfile

```diff
@@ -107,4 +107,71 @@ gh-all-prs:
   for project in "${projects[@]}"; do
     just --justfile {{justfile()}} gh-prs $project
     echo ""
-  done
\ No newline at end of file
+  done
+
+#
+# Heroku Commands
+#
+heroku-create-staging team pipeline:
+  #!/usr/bin/env bash
+  APP_NAME={{pipeline}}-staging
+  echo "Creating app '$APP_NAME' in team '{{team}}'..."
+  heroku pipelines:create {{pipeline}} --stage=staging --app=$APP_NAME --team={{team}}
+  heroku addons:create heroku-postgresql:essential-0 --app=$APP_NAME
+  heroku config:set ENVIRONMENT="staging" --app=$APP_NAME
+  heroku config:set DEBUG="True" --app=$APP_NAME
+
+heroku-create-production team pipeline:
+  #!/usr/bin/env bash
+  APP_NAME="{{pipeline}}-production"
+  echo "Creating app '$APP_NAME' in team '{{team}}'..."
+  heroku pipelines:create {{pipeline}} --stage=production --app=$APP_NAME --team={{team}}
+  heroku addons:create heroku-postgresql:standard-0 --app=$APP_NAME
+  heroku config:set ENVIRONMENT="production" --app=$APP_NAME
+  heroku config:set DEBUG="False" --app=$APP_NAME
+
+heroku-set-env-vars env_file='' app_name='':
+  #!/usr/bin/env bash
+
+  #
+  # Collects variables from a .env file and deploys.
+  #
+  # NOTES:
+  #  - This will overwrite existing variables! Confirm the correct variables are available.
+  #  - Because DB info is provisioned by Heroku, this script will ask the user if they 
+  #    want to purposefully include DB-related env vars.
+  #
+  function set_env() {
+      echo "Setting env vars from $env_file..."
+      [ -f "$env_file" ] && grep -vE '^(\s*$|#|DB_|REDIS_|ENVIRONMENT)' "$env_file" | tr '\n' ' ' | xargs heroku config:set -a $app_name || echo "Env file $env_file is not present, unable to set vars."
```

Comment:
This departs from the tn-gist version by also protecting against overwriting the `ENVIRONMENT` variable on the app.